### PR TITLE
f-rating@v0.8.0 - Simplify prop.

### PR DESCRIPTION
**Pull request by kevinrodrigues**
### Fixed
- Some comments from version 0.7.0.
- Tests so they work with new changes.
- Simplify prop `ratingDisplayType` so now only accepts 3 options.

### CODE REVIEW

File: packages/components/molecules/f-rating/src/components/Rating.vue [hunk 1]
```diff
@@ -69,7 +69,7 @@ export default {
         },
         ratingDisplayType: {
             type: String,
-            default: null,
+            required: true,
```
Comment (ashleynolan) [inline packages/components/molecules/f-rating/src/components/Rating.vue]: If this is required, wouldn't leaving it empty (to display no text) be impossible? Also, the README says that it's not required?

File: packages/components/molecules/f-rating/src/components/_tests/Rating.test.js [hunk 2]
```diff
@@ -174,18 +180,19 @@ describe('Rating', () => {
                     });
                 });
 
-                describe('and a rating type is set to `noRating`', () => {
+                describe('and a `reviewCount` is falsey', () => {
```
Comment (ashleynolan) [inline packages/components/molecules/f-rating/src/components/_tests/Rating.test.js:183]: Do you need a test here for if `ratingDisplayType` is null? (depends if it's required or not)

File: packages/components/molecules/f-rating/src/components/Rating.vue (hunk 1 above)
Comment (kevinrodrigues) [reply packages/components/molecules/f-rating/src/components/Rating.vue]: That's a good point. Will have a think over this. 👍

File: packages/components/molecules/f-rating/src/components/Rating.vue [hunk 3]
```diff
@@ -126,20 +126,24 @@ export default {
...
         /**
          * Gets the correct rating display format from translations.
          *
-         * @todo - If the component is using `short` as a `starRatingSize` we shouldn't display text
-         * alongside it for now. (TBC with design - ticket in backlog).
+         * 1. If reviewCount is Zero, return "No ratings yet".
+         * 2. If reviewCount is available, return `ratingDisplayType` .
          *
-         * @returns {string}
+         * @returns {String}
```
Comment (zlatinivanov-je) [inline packages/components/molecules/f-rating/src/components/Rating.vue]: Is the case change necessary? Seems to break consistency from above

**Review by zlatinivanov-je — COMMENTED** [review]
LGTM 
Subject to comments

File: packages/components/molecules/f-rating/src/components/_tests/Rating.test.js (hunk 2 above)
Comment (kevinrodrigues) [reply packages/components/molecules/f-rating/src/components/_tests/Rating.test.js:183]: I think this one is covered [here](https://github.com/justeat/fozzie-components/pull/2194/files#diff-bd17adcec33ca1bc737ba12a318b084ea668c55175957ffab8198ceeba031481R419).

File: packages/components/molecules/f-rating/src/components/Rating.vue (hunk 3 above)
Comment (kevinrodrigues) [reply packages/components/molecules/f-rating/src/components/Rating.vue]: My bad.

File: packages/components/molecules/f-rating/src/components/Rating.vue (hunk 1 above)
Comment (kevinrodrigues) [reply packages/components/molecules/f-rating/src/components/Rating.vue]: Fixed.

**Review by dawidchar — APPROVED** [review]

**Review by ashleynolan — APPROVED** [review]
